### ISSUE TITLE
batch size training versus validation

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_multilingual_text_classification#75]
why should we use different values ?

**Comment by tarrade** [comment]
https://forums.fast.ai/t/different-batch-size-for-train-and-valid-data-loaders/30691

you don’t compute the gradients for the validation set, so you have more GPU memory available. Usually it’s even bs*2 vor the validation set.
